src/site/content/en/blog/badging-api/index.md

```diff
@@ -159,9 +154,57 @@ Don't assume anything about how the user agent wants to display the badge.
 Some user agents may take a number like "4000" and rewrite it as
 "99+". If you saturate the badge yourself (for example by setting it to "99")
 then the "+" won't appear. No matter the actual number, just set
-`Badge.set(unreadCount)` and let the user agent deal with displaying it
+`setAppBadge(unreadCount)` and let the user agent deal with displaying it
 accordingly.
 
+While the Badging API *in Chrome* requires an installed app with an icon
+that can be badged, you shouldn't make calls to the Badging API dependent
+on the install state. Just call the API when it exists. If it works,
+it works. If not, it simply doesn't.
+
+{% Aside %}
+  Once the origin trial is completed, the API should switch to
+  `navigator.setAppBadge()`.
+{% endAside %}
+
+## Supporting both the first and second origin trial {: #backwards-compat }
+
+In Chrome 79, the first origin trial ended, and based on developer feedback
+received during the first origin trial, was replaced by a new origin trial
+that uses a slightly different API design. Unfortunately, it is not backwards
+compatible, and calls to the previous API will fail.
+
+If you have an existing implementation, and want to provide backwards
+compatibility for your users, you can wrap the calls in the following
+wrapper function. Once all of your users have migrated to Chrome 79+, you
+can remove this wrapper function from you code.
+
+```js
+function setBadge(...args) {
+  if (navigator.setExperimentalAppBadge) {
```

Comment:
Is it worth explaining why this unholiness is necessary?
(I'm so sorry)

Reply:
I think it's too much of an aside that gets into the weeds of JavaScript vs how to use the badging api. Willing to hear push back though if you think it's important.

Reply:
Nah, SGTM :)